server/controllers/UserController.js

```diff
@@ -62,3 +62,6 @@ exports.delete = async (req, res) => {
   await User.destroy({ where: { id } });
   res.status(204).send();
 };
+
+
+// send request to the route postman
```

Comment:
whats this for?

Reply:
this line was supposed to be deleted. fixed it.